backend/protos/xyz/block/ftl/v1/console/console.proto

```diff
@@ -170,6 +170,10 @@ message EventsQuery {
     optional string dest_verb = 2;
     optional string source_module = 3;
   }
+  message ModuleFilter {
```

Comment:
Or we could revisit this when we refactor the query into separate targeted ones

Reply:
This might be a longer discussion that for this PR. We should chat about this setup and how we might want it to work in the future. It's mostly designed at this point for the `Timeline` `Events` page, but could def use some more attention for future features.